src/Filter/CheckboxFilter/CheckboxFilter.stories.tsx

```diff
@@ -40,17 +40,7 @@ export const Default = {
       "B",
       "Gate 1"
     ],
-    value: ["A", "b"],
-    variant: "popper"
-  },
-
-  render: Template
-};
-
-export const AlwaysOpen = {
-  args: {
-    ...Default.args,
-    variant: "always-open"
+    value: ["A", "b"]
```

Comment:
As suggested by Matt, `always open autocomplete` has been removed as it is no longer used in Virto.